### ISSUE TITLE
Add Jekyll Liquid language

### ISSUE THREAD

**Issue by jayvdb** [coala/coala#5668]
We use Liquid language in Jekyll sites.

We should have a language definition for it.

**Comment by jayvdb** [comment]
A fun one I ran into is it doesnt appear possible to use `{{` inside a `{{`.

e.g. `{{ foo | replace: "{{}}", "replacement" }}` is a syntax error

{% assign foo2 = foo | replace: "{{}}", "replacement" %}` works ok

**Comment by kunalvaswani123** [comment]
Could you explain this issue to me?

**Comment by shashwatmishra76** [comment]
@jayvdb I guess this issue just requires me to add a language definition for Jekyll the same way I did for various shells.Can I be assigned to this please?

**Comment by jayvdb** [comment]
@shashwatmishra76 , yup, after you finished the other issue, you can be assigned to this issue.
If you dont have any specific interest in jekyll, there are similar issues for SCSS and SASS, which are higher priority because we have bears for those languages, but the language definition is missing.

**Comment by shashwatmishra76** [comment]
@jayvdb Okay.I will be doing the higher priority ones first.

**Comment by jayvdb** [comment]
Assigned to @kunalvaswani123

**Comment by rajdeepbharati** [comment]
Can I work on this?

**Comment by Anmolbansal1** [comment]
can I work on this @jayvdb @li-boxuan

**Comment by SimonZYC** [comment]
Hi @Anmolbansal1 , are you still working on this issue? I'd like to take it as well.
